generators/getMetric.js

```diff
@@ -0,0 +1,68 @@
+const path = require('path')
+const fs = require('fs')
+const data = require('./getMetricFunctions.js')
```

Comment:
is `data` an appropriate name here? Below you have the ambiguous `data.data`

Reply:
Changed it to objects, it is in fact an array of objects, representing the different metrics